content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -486,6 +486,7 @@ Here are some examples of device plugin implementations:
 * [Akri](https://github.com/project-akri/akri), which lets you easily expose heterogeneous leaf devices (such as IP cameras and USB devices).
 * The [AMD GPU device plugin](https://github.com/ROCm/k8s-device-plugin)
 * The [generic device plugin](https://github.com/squat/generic-device-plugin) for generic Linux devices and USB devices
+* [HAMi](https://github.com/Project-HAMi/HAMi),Heterogeneous AI Computing Virtualization Middleware,Support unified scheduling, management, and monitoring of heterogeneous AI chips(such as Nvidia,Cambricon,Hygon,Iluvatar,Mthreads,Ascend,Metax)
```

Comment:
```suggestion
* [HAMi](https://github.com/Project-HAMi/HAMi), formerly known as 'k8s-vGPU-scheduler', is a
  heterogeneous device management middleware for Kubernetes. It supports diverse AI chips,
  including Nvidia, Cambricon, Hygon, Iluvatar, Mthreads, Ascend, and Metax.
```

Reply:
Do not mark it as 'Resolved' if any differing opinions. We value and encourage diverse ideas! 😅

Reply:
It should be a mistake, not intentional. I have simplified the description, can you help review it again? Thank you.